packages/labs/context/src/lib/value-notifier.ts

```diff
@@ -57,8 +57,11 @@ export class ValueNotifier<T> {
           this.callbacks.delete(callback);
         });
       }
+      const dispose = this.callbacks.get(callback)!;
+      callback(this.value, dispose);
+    } else {
+      callback(this.value);
```

Comment:
Nit: to be more consistent with the specifier used in rest of file.
```suggestion
      const disposer = this.callbacks.get(callback)!;
      callback(this.value, disposer);
    } else {
      callback(this.value);
```

Reply:
Nice!